fenix/app/metrics.yaml

```
@@ -2539,6 +2539,28 @@ metrics:
     metadata:
       tags:
         - Experiments
+  device_total_ram:
+    type: quantity
+    lifetime: application
+    description: >
+      The total amount of memory this device has GBs, when available will be
+      MemoryInfo.advertisedMem otherwise it will be MemoryInfo.totalMem
```

```suggestion
      The total amount of memory this device has in gigabytes. When available will be
      MemoryInfo.advertisedMem otherwise it will be MemoryInfo.totalMem. 
      This doesn't represent memory available to the application however.
```

Fixed a nit and also added a clarification (see overall note).

---

👍🏽